### ISSUE TITLE
CIFAR-10 example

### ISSUE THREAD

**Issue by ajherman** [tensorflow/cleverhans#219]
When I run ex_cifar10_tf.py example, the model appears not to train.  That is, the printed test accuracy is 0.1 after every epoch of (non-adversarial) training.  Perhaps I am misunderstanding what this script is intended to do?

**Comment by npapernot** [comment]
It may be the case that the learning rate is not appropriate now that we changed the optimizer used by model_train. Can you try to modify it and let us know if it works?

**Comment by npapernot** [comment]
Nevermind I just saw your PR that does just that. Thanks!
